### DISCUSSION TITLE
Discussion topics for the Monday June 28 Developer Sync (1pm ET)

### DISCUSSION THREAD

**Discussion by jimduff-usds** [CDCgov/prime-reportstream#1423] [Ideas]
@mauricereeves
@Mr0grog
@mry-usds
@RickHawesUSDS
@ronaldheft-usds
@joemeree
@tlosbo
@pathikkothari
@td-usds
@pablocaviglia

Please enter your discussion topics here.

- @td-usds  - hoping you'll be able to cover your Docker change proposal.
- @ronaldheft-usds would like to cover the IP address CIDR proposal: https://github.com/CDCgov/prime-reportstream/pull/1407
- I'd like to cover our position on mobile apps connecting directly to ReportStream to upload

Other topics?

**Comment by td-usds** [comment]
re: docker build proposal: not yet, I need to do some more work on it first. Probably next week?

**Comment by jimduff-usds** [comment]
We also have this giant discussion, which I'm happy to defer as well:
2. (@carlosfelix2 ) _BLANK types and cardinality.  (heh heh:  check out https://github.com/CDCgov/prime-reportstream/issues/230)
3.

**Comment by carlosfelix2** [comment]
New end to end compare test
